Build/Testbed.vcxproj

```diff
@@ -224,6 +224,8 @@
     <ClInclude Include="..\Testbed\imgui\stb_textedit.h" />
     <ClInclude Include="..\Testbed\imgui\stb_truetype.h" />
     <ClInclude Include="resource.h" />
+    <ClInclude Include="Scene2Settings.h" />
+    <ClInclude Include="Scene2Simulation.h" />
```

Comment:
Lets not forget to put these files on `"..\Testbed\Tests\SVQA\Scene2\Scene2...h"`. We need to manually create a file on the directory ` ..\Testbed\Tests\SVQA\Scene2`, then add it into Visual Studio. Why? Cuz VS sucks :(